tests/fixtures/cgweb_orders/microbial_fastq.json

```diff
@@ -0,0 +1,35 @@
+{
+    "name": "Microbial Fastq order",
+    "customer": "cust002",
+    "comment": "",
+    "samples": [
+        {
+            "application": "WGSPCFC060",
+            "comment": "sample comment",
+            "container": "Tube",
+            "container_name": "prov1",
+            "data_analysis": "fastq",
+            "data_delivery": "fastq",
+            "elution_buffer": "Nuclease-free water",
+            "name": "prov1",
+            "priority": "priority",
+            "sex": "male",
+            "volume": "1",
+            "well_position": ""
+        },
+        {
+            "application": "WGSPCFC060",
+            "comment": "sample comment",
+            "container": "Tube",
+            "container_name": "prov2",
+            "data_analysis": "fastq",
+            "data_delivery": "fastq",
+            "elution_buffer": "Nuclease-free water",
+            "name": "prov2",
+            "priority": "priority",
+            "sex": "male",
+            "volume": "2",
+            "well_position": ""
+        }
+    ]
+}
```

Comment:
@Vince-janv I took a fastq order and modified it a bit, could you have a look whether this is correct here? Then I will add tests around this.